app/views/content/what-pension-does-a-teacher-get.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "About the teachers' pension scheme"
+heading: "About the teachers' pension scheme"
+description: |-
+  The teachers' pension is one of the most generous in the country. Learn more about the benefits of the pension scheme once you become a teacher.
+date: "2022-11-29"
+image: false
+promo_content:
+    - content/train-to-be-a-teacher/promos/mailing-list-promo-salaries
+related_content:
+    Salaries and benefits : "/salaries-and-benefits"
+    Steps to become a teacher : "/steps-to-become-a-teacher"
+    Train to be a teacher if you have or are studying for a degree : "/train-to-be-a-teacher/if-you-have-a-degree"
+    Scholarships and bursaries for training to teach : "/funding-and-support/scholarships-and-bursaries"
+keywords:
+  - Teachers pension
+  - Pension
+  - Benefits
+  - Earnings
+---
+One of the great benefits of a teaching career is a secure pension that will help you save for your future. 
+
+The teachers’ pension scheme is one of the most generous in the country. These are some of the key benefits of the scheme.
+
+## Easy to get started
+
+As a teacher, you’ll be automatically enrolled at the start of your career, and remain in the scheme when moving teaching jobs, unless you opt out. 
+
+Under the scheme, you'll automatically pay from your salary into your pension each month. At the same time, your employer also contributes.
+
+Once you join the scheme, it's quick and easy to manage your pension online.
+
+## Based on your salary
+
+It’s a ‘defined benefit’ pension, which means that it’s based on your salary rather than the amount you contribute.
+
+If your salary increases, the amount you and your employer pay will increase too. You can also choose to pay extra contributions if you want to. Learn more about paying extra contributions on the [scheme website](https://www.teacherspensions.co.uk/members/working-life/paying-in/increasing-your-pension/additional-pension.aspx) (opens in a new window).
```

Comment:
```suggestion
If your salary increases, the amount you and your employer pay will increase too. You can also choose to pay extra contributions if you want to. [Learn more about paying extra contributions](https://www.teacherspensions.co.uk/members/working-life/paying-in/increasing-your-pension/additional-pension.aspx) (opens in a new window).
```

@philspray I don't think `scheme website` gives enough context, particularly for screen reader users skipping through links, so would suggest you change along these lines (doesn't have to be exactly this!)?